jsonnet/mixin/alerts/alerts.libsonnet

```diff
@@ -104,6 +104,25 @@
           },
         ],
       },
+      {
+        name: 'config-reloaders',
+        rules: [
+          {
+            alert: 'ConfigReloaderSidecarErrors',
+            expr: |||
+              (min by (job,pod) (reloader_last_reload_successful{job=~"%(prometheusSelector)s|%(alertManagerSelector)s"}) == 0)
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              description: 'Errors encountered while the {{$labels.pod}} config-reloader sidecar attempts to sync config in {{$labels.namespace}} namespace.',
```

Comment:
since you used `min by (job,pod) (...`, the namespace label wouldn't be present in the query result :)

Reply:
I can add it back in and update the selector